README.md

```diff
@@ -45,94 +45,46 @@ For more information about Daffodil, see the [Website].
 * C compiler C99 or higher
 * Mini-XML Version 3.2 or higher
 
-Since Daffodil has a DFDL to C backend, you will need a C compiler
-([gcc] or [clang]), the [Mini-XML] library, and possibly the GNU
-[argp] library if your system's C library doesn't include it.  You can
-install gcc and libmxml as system packages on most Unix based
-platforms with distribution-specific packager commands such as (Debian
-and Ubuntu):
-
-    # Just mentioning all other packages you might need too
-    sudo apt install build-essential curl git libmxml-dev
-
-You will need the Java Software Development Kit ([JDK]) and the Scala
-Build Tool ([SBT]) to build Daffodil, run all tests, create packages,
-and more.  [SDK] offers an easy and uniform way to install both java
-and sbt on any Unix based platform:
-
-    curl -s "https://get.sdkman.io" | bash
-    sdk install java
-    sdk install sbt
-
-You can edit the Compile / cCompiler setting in build.sbt if you don't
-want sbt to call your C compiler with "cc" as the driver command.
-
-On Windows, the easiest way to install gcc and libargp is to install
-[MSYS2]'s collection of free tools and libraries although MSYS2 has no
-package for libmxml which you'll need to build from source.  First
-install [MSYS2] following its website's installation instructions,
-then run the following commands in a "MSYS2 MSYS" window:
-
-    pacman -S gcc git libargp-devel make pkgconf
-    git clone https://github.com/michaelrsweet/mxml.git
-    cd mxml
-    ./configure --prefix=/usr --disable-shared --disable-threads
-    make
-    make install
-
-You also need to install [JDK} and [SBT] from their Windows
-installation packages and define an environment variable using
-Windows' control panel for editing environment variables.  Define an
-environment variable with the name `MSYS2_PATH_TYPE` and the value
-`inherit`.  Now when you open a new "MSYS2 MSYS" window from the Start
-Menu, you will be able to type your sbt commands in the MSYS2 window
-and both sbt and daffodil will be able to call the C compiler.
+See [BUILD.md](BUILD.md) for more details.
 
 ## Getting Started
 
-Below are some of the more common commands used for Daffodil development.
+[SBT] is the officially supported tool to build Daffodil.  Here are
+some of the more commonly used commands for Daffodil development.
 
-### Compile
+* Compile source code:
```

Comment:
I personally perfer the ### headings over the bullets.  It makes it easier to skim over and find the command you might need, rather than having the full sentences.

Reply:
OK, fair enough.  I'll put the ### headings back.